components/cluster/command/display.go

```diff
@@ -108,7 +115,21 @@ func displayDashboardInfo(clusterName string) error {
 	}
 
 	u.Path = "/dashboard/"
-	fmt.Println(u.String())
+
+	if tlsCfg != nil {
+		fmt.Println(
```

Comment:
Can you combine the two `fmt.Println`?

Reply:
It could be better formatting by printing those content line by line instead of using a combined multi-line template, `go fmt` would force adding indent to the first line.